raiwidgets/setup.py

```diff
@@ -17,8 +17,9 @@
 
 # Use requirements.txt to set the install_requires
 with open('requirements.txt') as f:
+    # Note: use ["responsibleai==%s" % version] eventually
     install_requires = [line.strip() for line in f] + \
-        ["responsibleai==%s" % version]
+        ["responsibleai==0.0.3"]
```

Comment:
Why are we using different versioning for this package while still using the same versioning in CHANGES.md?

Reply:
We should not change this, responsible ai and rai widget have version aligned.